### PR TITLE
Manager functionallity

### PR DESCRIPTION
**Pull request by FunnyBytes**
Refactored and Rename MenuInitializer class

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/Espress0JavaTeam/AirLine/pull/5" target="_blank">ReviewNinja

File: workspace/AirLine/.classpath [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (VolodymyrBondarchuk) [inline workspace/AirLine/.classpath]: it is not necessary to commit

File: workspace/AirLine/.gitignore [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Output
```
Comment (VolodymyrBondarchuk) [inline workspace/AirLine/.gitignore]: it is not necessary to commit

File: workspace/AirLine/.gitignore [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+out/
```
Comment (VolodymyrBondarchuk) [inline workspace/AirLine/.gitignore]: it is not necessary to commit

File: workspace/AirLine/.project [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (VolodymyrBondarchuk) [inline workspace/AirLine/.project]: it is not necessary to commit

File: workspace/AirLine/readme.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+#Here should be description of the project
```
Comment (VolodymyrBondarchuk) [inline workspace/AirLine/readme.md]: it is not necessary to commit

File: workspace/AirLine/src/com/acme/command/Command.java [hunk 6]
```diff
@@ -0,0 +1,7 @@
+package com.acme.command;
```
Comment (VolodymyrBondarchuk) [inline workspace/AirLine/src/com/acme/command/Command.java]: this file already exist in project. Why do not commit it?

File: workspace/AirLine/src/com/acme/command/impl/destination/PrintDestinations.java [hunk 7]
```diff
@@ -0,0 +1,25 @@
+package com.acme.command.impl.destination;
```
Comment (VolodymyrBondarchuk) [inline workspace/AirLine/src/com/acme/command/impl/destination/PrintDestinations.java]: this file already exist in project. Why do not commit it?

File: workspace/AirLine/src/com/acme/command/impl/flight/BuyTicket.java [hunk 8]
```diff
@@ -0,0 +1,42 @@
+package com.acme.command.impl.flight;
```
Comment (VolodymyrBondarchuk) [inline workspace/AirLine/src/com/acme/command/impl/flight/BuyTicket.java]: this file already exist in project. Why do not commit it?

File: workspace/AirLine/src/com/acme/command/impl/flight/PrintFlights.java [hunk 9]
```diff
@@ -0,0 +1,26 @@
+package com.acme.command.impl.flight;
```
Comment (VolodymyrBondarchuk) [inline workspace/AirLine/src/com/acme/command/impl/flight/PrintFlights.java]: this file already exist in project. Why do not commit it?

File: workspace/AirLine/src/com/acme/command/impl/plane/AddPlane.java [hunk 10]
```diff
@@ -0,0 +1,38 @@
+package com.acme.command.impl.plane;
```
Comment (VolodymyrBondarchuk) [inline workspace/AirLine/src/com/acme/command/impl/plane/AddPlane.java]: this file already exist in project. Why do not commit it?

**Comment by FunnyBytes** [comment]
fixed.

**Pull request closed without merging**